modules/large-deployments/pages/hub-auth.adoc

```diff
@@ -113,30 +113,30 @@ The map has two entries:
 import xmlrpclib
 
 
-HUB_URL = "http://localhost:8888/hub/rpc/api"
```

Comment:
Would it make sense to leave an example (working) URL for starters?

Reply:
I guess it would help, yes